### PR TITLE
Replace window.require usage with import

### PR DESCRIPTION
**Pull request by paninaro**
Based on @ChiaMineJP's findings from https://github.com/Chia-Network/chia-blockchain/pull/12523

### CODE REVIEW

**Review by ChiaMineJP — APPROVED** [review]
I've confirmed that this PR works in combination with https://github.com/Chia-Network/chia-blockchain/pull/12523 .
In order to get the best result from this PR,
This PR should be
- merged into `main` and
- `chia-blockchain`'s `main` branch points to the latest GUI ref

before merging [12523](https://github.com/Chia-Network/chia-blockchain/pull/12523) to `chia-blockchain`'s `main`.
By doing this, the whole `packages/gui/node_modules` can be excluded from `app.asar` and this reduces more file size of `app.asar`.

**Comment by seeden** [comment]
LGTM

File: packages/gui/src/components/app/AppProviders.tsx [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
+  LayoutLoading,
+  dark,
+  light,
+  ErrorBoundary,
+} from '@chia/core';
 import { store, api } from '@chia/api-react';
 import { Trans } from '@lingui/macro';
 import { i18n, defaultLocale, locales } from '../../config/locales';
 import AppState from './AppState';
+import { WebSocket as WS } from 'ws';
```
Comment (seeden) [inline packages/gui/src/components/app/AppProviders.tsx:20]: WebSocket is default import I think
import WebSocket from 'ws';

**Review by seeden — COMMENTED** [review]
I wrote one change related to WS documentation

File: packages/gui/src/components/app/AppProviders.tsx (hunk 1 above)
Comment (ChiaMineJP) [reply packages/gui/src/components/app/AppProviders.tsx:20]: I investigated it.

https://github.com/websockets/ws/blob/master/index.js#L10

```javascript
'use strict';

const WebSocket = require('./lib/websocket');

WebSocket.createWebSocketStream = require('./lib/stream');
WebSocket.Server = require('./lib/websocket-server');
WebSocket.Receiver = require('./lib/receiver');
WebSocket.Sender = require('./lib/sender');

WebSocket.WebSocket = WebSocket; // <-- Both default import and named import are correct
WebSocket.WebSocketServer = WebSocket.Server;

module.exports = WebSocket;
```

Comment (seeden) [reply packages/gui/src/components/app/AppProviders.tsx:20]: Thanks, I will check our version and build

**Pull request merged**